src/services/geometry/richgeo/IrtGeoDRICH.h

```diff
@@ -13,7 +13,7 @@ namespace richgeo {
       IrtGeoDRICH(gsl::not_null<const dd4hep::Detector*> det_, gsl::not_null<const dd4hep::rec::CellIDPositionConverter*> conv_, std::shared_ptr<spdlog::logger> log_) :
         IrtGeo("DRICH",det_,conv_,log_) { DD4hep_to_IRT(); }
       ~IrtGeoDRICH();
-
+    TVector3 GetSensorSurfaceNorm(CellIDType);
```

Comment:
Can this be private only?

Reply:
No. Alternatively one can move this method's implementation to wherever it is needed; currently the only user is https://github.com/eic/drich-dev/pull/112.

Reply:
Shall I resolve this conversation?